### PR TITLE
Added html and css codes.. Added Form for collecting data

### PR DESCRIPTION
**Pull request by daviidy**
Worked on all the requirements given on this project:

- Used semantic HTML elements for the header, the main content, the form and the footer.
- Styled the elements to look like the real webpage.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Sign Up Page</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/016a3b4736.js" type="text/javascript"></script>
```
Comment (arslanbisharat) [inline index.html]: The type attribute is unnecessary for JavaScript resources.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              </fieldset>
+              <fieldset>
+                <label for="email">Phone (recommended)</label>
+                <input id="email" type="text" name="email" value="">
+                <span class="span-input">Standard call, messaging or data rates may apply.</span>
+              </fieldset>
+              <fieldset>
+                <label for="email">Password</label>
+                <input id="email" type="text" name="email" value="">
+              </fieldset>
```
Comment (arslanbisharat) [inline index.html]: Duplicate ID email

File: style.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  margin: 0;
+  padding: .5rem;
+  background-color: #eceef1
+}
+*{
+  box-sizing: border-box;
+}
+
+header{
+  /* margin-top: .3rem; */
```
Comment (arslanbisharat) [inline style.css]: Please remove commented code

File: style.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
+body{
+  font-family: 'Montserrat', sans-serif;
+  margin: 0;
+  padding: .5rem;
+  background-color: #eceef1
+}
+*{
+  box-sizing: border-box;
+}
```
Comment (arslanbisharat) [inline style.css]: Please follow the order of selection

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes :flushed:
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard. You are there._:smile:
**Please have a look at the suggestions to get more understanding.** 
- [ ]  Your readme is not descriptive. Please make it more descriptive.

- [ ] Remove linter errors

- [ ] Please follow the order of selection in the CSS file

- [ ] Remove duplicate ID email from line 58 and 61

- [ ] The type attribute is unnecessary for JavaScript resources. Please remove it.

**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+              <a href="#">Learn more</a>
+            </p>
+            <form class="form">
+              <fieldset>
+                <label for="email">Email address</label>
+                <input id="email" type="email" name="email" value="">
+              </fieldset>
+              <fieldset>
+                <label for="phone">Phone (recommended)</label>
+                <input id="phone" type="text" name="phone" value="">
```
Comment (kevcap) [inline index.html]: [OPTIONAL] 
It should be` type="tel"`. Please fix that.

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

## Congratulations team! Good job making my previously requested changes, thanks for putting your effort into building it! Now you can merge this pull request and move on to your next awesome project!

## [OPTIONAL CHANGES]
- [ ] Add a live demo of your project in your readme.
**⚠These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
- [ ] The type of the phone input should be `type="tel"` instead of text. Please fix that.



## As always, I wish you all the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

File: index.html (hunk 5 above)
Comment (cyonii) [reply index.html]: I've fixed it. Thank you for the tip

**Pull request closed without merging**